python/lsst/daf/persistence/butler.py

```diff
@@ -1567,6 +1589,29 @@ def _resolveDatasetTypeAlias(self, datasetType):
 
         return datasetType
 
+    def cfgNeedsFixup(self, cfg):
+        """check the passed-in cfg and see if it has a path to an Old Butler repository. If so, return true (Old
+        Butler repositories should be referred to by nested repositoryCfg in the parents list)."""
+        if cfg.parents:
+            for i, parent in enumerate(cfg.parents):
+                if isinstance(parent, RepositoryCfg):
+                    continue
+                parentCfg, isOldButlerRepository = self._getRepositoryCfg(parent)
+                if isOldButlerRepository:
+                    cfg._parents[i] = cfg._normalizeParents(cfg.root, [parentCfg])[0]
+
+
+def checkCfgs(cfgList):
```

Comment:
It's not clear to me where this method gets used: it's not used in this commit anywhere. Also, it has no docstring.